src/main/java/uk/gov/hmcts/reform/iacasedocumentsapi/domain/entities/ClarifyingQuestionAnswer.java

```diff
@@ -0,0 +1,55 @@
+package uk.gov.hmcts.reform.iacasedocumentsapi.domain.entities;
+
+import java.util.List;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+import uk.gov.hmcts.reform.iacasedocumentsapi.domain.entities.ccd.field.Document;
+import uk.gov.hmcts.reform.iacasedocumentsapi.domain.entities.ccd.field.IdValue;
+
```

Comment:
Could possibly introduce lombok `@Getter` and `@Setter` annotations, but happy to leave as is.

Reply:
@bradsorour  good catch, i will add it

Reply:
This is now added.